src/thor/timedistancematrix.cc

```diff
@@ -141,416 +203,217 @@ void TimeDistanceMatrix::ExpandForward(GraphReader& graphreader,
   if (!from_transition && nodeinfo->transition_count() > 0) {
     const NodeTransition* trans = tile->transition(nodeinfo->transition_index());
     for (uint32_t i = 0; i < nodeinfo->transition_count(); ++i, ++trans) {
-      ExpandForward(graphreader, trans->endnode(), pred, pred_idx, true);
+      Expand(graphreader, trans->endnode(), pred, pred_idx, true);
     }
   }
 }
 
-// Calculate time and distance from one origin location to many destination
-// locations.
-std::vector<TimeDistance>
-TimeDistanceMatrix::OneToMany(const valhalla::Location& origin,
-                              const google::protobuf::RepeatedPtrField<valhalla::Location>& locations,
-                              GraphReader& graphreader,
-                              const sif::mode_costing_t& mode_costing,
-                              const travel_mode_t mode,
-                              const float max_matrix_distance,
-                              const uint32_t matrix_locations) {
-  // Set the mode and costing
-  mode_ = mode;
-  costing_ = mode_costing[static_cast<uint32_t>(mode_)];
-  current_cost_threshold_ = GetCostThreshold(max_matrix_distance);
-
-  // Construct adjacency list, edge status, and done set. Set bucket size and
-  // cost range based on DynamicCost. Initialize A* heuristic with 0 cost
-  // factor (needed for setting the origin).
-  astarheuristic_.Init({origin.ll().lng(), origin.ll().lat()}, 0.0f);
```

Comment:
I removed all references to AStarHeuristic, this is a Dijkstra, no need for that